Flabbergasted-Flamingos/Bangazon-admin-app
# Rb styles

# Description
This is 2 style sheets to syle all the pages so that this thing is better lookin


## Related Ticket(s)
This affects all the tickets

## Problem to Solve
Forms and lists are easier on the eyes.

## Proposed Changes
All templates had to be tweaked to make things consistent across the app. Included nav bar in all elements and styles.

## Expected Behavior
All behaviour should stay the exact same, PLEASE PLAY AROUND WITH ALL THE DIFFERENT PAGES

## Steps to Test Solution

1. 
```
git fetch --all
git checkout rb_styles
pyman runserver
```
2. Play with everything and make sure all forms work, details views show up

## Testing

[x ] I certify that all existing tests pass
